src/domain/registration/form/RegistrationFormUtils.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  RegistrationFormValues,
+  SubmitChildValues,
+} from '../types/RegistrationTypes';
+
+/**
+ * Convert FormValues from RegistrationForm to childMutation
+ * @param {RegistrationFormValues} stateFormValues Values from state.
+ * @returns {HomeFormValues} initialValues used in form.
+ */
+export const convertFormValuesToSubmitChildMutation = (
```

Comment:
Why we even need this? Data in redux store is already beautified for sending

Reply:
is it backend var required?

Reply:
True! Deleted!